### PR TITLE
🌐 Update Chinese translations with new source files

### PR DESCRIPTION
**Pull request by mahone3297**
The docs in language zh is out of date.
So the high line is wrong on the python code, that makes the reader confuse.

I copy the py-code from language en to zh.

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 4efcc6911151e5ac07f048e304023c4197925b88 at: https://64977ec294f5d8307aebe55d--fastapi.netlify.app

**Review by wdh99 — APPROVED** [review]

File: docs/zh/docs/tutorial/body-fields.md [hunk 1]
```diff
@@ -6,9 +6,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="4"
+    {!> ../../../docs_src/body_fields/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-fields.md:30]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-fields.md [hunk 2]
```diff
@@ -6,9 +6,41 @@
...
+
+    ```Python hl_lines="2"
+    {!> ../../../docs_src/body_fields/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-fields.md:39]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 3]
```diff
@@ -8,9 +8,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="19-21"
+    {!> ../../../docs_src/body_multiple_params/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:32]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 4]
```diff
@@ -8,9 +8,41 @@
...
+
+    ```Python hl_lines="17-19"
+    {!> ../../../docs_src/body_multiple_params/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:41]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 5]
```diff
@@ -72,9 +112,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="24"
+    {!> ../../../docs_src/body_multiple_params/tutorial003_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:136]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 6]
```diff
@@ -72,9 +112,41 @@
...
+
+    ```Python hl_lines="20"
+    {!> ../../../docs_src/body_multiple_params/tutorial003_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:145]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 7]
```diff
@@ -109,9 +181,41 @@ q: str = None
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="28"
+    {!> ../../../docs_src/body_multiple_params/tutorial004_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:205]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 8]
```diff
@@ -109,9 +181,41 @@ q: str = None
...
+
+    ```Python hl_lines="25"
+    {!> ../../../docs_src/body_multiple_params/tutorial004_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:214]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 9]
```diff
@@ -131,9 +235,41 @@ item: Item = Body(embed=True)
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="18"
+    {!> ../../../docs_src/body_multiple_params/tutorial005_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:259]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/body-multiple-params.md [hunk 10]
```diff
@@ -131,9 +235,41 @@ item: Item = Body(embed=True)
...
+
+    ```Python hl_lines="15"
+    {!> ../../../docs_src/body_multiple_params/tutorial005_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/body-multiple-params.md:268]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/cookie-params.md [hunk 11]
```diff
@@ -6,9 +6,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="3"
+    {!> ../../../docs_src/cookie_params/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/cookie-params.md:30]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/cookie-params.md [hunk 12]
```diff
@@ -6,9 +6,41 @@
...
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/cookie_params/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/cookie-params.md:39]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/cookie-params.md [hunk 13]
```diff
@@ -17,9 +49,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="10"
+    {!> ../../../docs_src/cookie_params/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/cookie-params.md:73]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/cookie-params.md [hunk 14]
```diff
@@ -17,9 +49,41 @@
...
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/cookie_params/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/cookie-params.md:82]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/extra-data-types.md [hunk 15]
```diff
@@ -55,12 +55,76 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="1  3  13-17"
+    {!> ../../../docs_src/extra_data_types/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/extra-data-types.md:79]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/extra-data-types.md [hunk 16]
```diff
@@ -55,12 +55,76 @@
...
+
+    ```Python hl_lines="1  2  11-15"
+    {!> ../../../docs_src/extra_data_types/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/extra-data-types.md:88]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/extra-data-types.md [hunk 17]
```diff
@@ -55,12 +55,76 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="19-20"
+    {!> ../../../docs_src/extra_data_types/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/extra-data-types.md:117]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/extra-data-types.md [hunk 18]
```diff
@@ -55,12 +55,76 @@
...
+
+    ```Python hl_lines="17-18"
+    {!> ../../../docs_src/extra_data_types/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/extra-data-types.md:126]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 19]
```diff
@@ -6,19 +6,83 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="3"
+    {!> ../../../docs_src/header_params/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:30]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 20]
```diff
@@ -6,19 +6,83 @@
...
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/header_params/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:39]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 21]
```diff
@@ -6,19 +6,83 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="10"
+    {!> ../../../docs_src/header_params/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:72]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 22]
```diff
@@ -6,19 +6,83 @@
...
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/header_params/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:81]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 23]
```diff
@@ -44,9 +108,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="12"
+    {!> ../../../docs_src/header_params/tutorial002_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:132]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 24]
```diff
@@ -44,9 +108,41 @@
...
+
+    ```Python hl_lines="8"
+    {!> ../../../docs_src/header_params/tutorial002_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:141]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 25]
```diff
@@ -62,9 +158,50 @@
...
+
+    ```Python hl_lines="7"
+    {!> ../../../docs_src/header_params/tutorial003_py310.py!}
+    ```
+
+=== "Python 3.9+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:191]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/header-params.md [hunk 26]
```diff
@@ -62,9 +158,50 @@
...
+
+    ```Python hl_lines="9"
+    {!> ../../../docs_src/header_params/tutorial003_py39.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/header-params.md:200]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/path-params-numeric-validations.md [hunk 27]
```diff
@@ -6,19 +6,83 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="3-4"
+    {!> ../../../docs_src/path_params_numeric_validations/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/path-params-numeric-validations.md:30]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/path-params-numeric-validations.md [hunk 28]
```diff
@@ -6,19 +6,83 @@
...
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/path_params_numeric_validations/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/path-params-numeric-validations.md:39]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/path-params-numeric-validations.md [hunk 29]
```diff
@@ -6,19 +6,83 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="11"
+    {!> ../../../docs_src/path_params_numeric_validations/tutorial001_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/path-params-numeric-validations.md:72]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/path-params-numeric-validations.md [hunk 30]
```diff
@@ -6,19 +6,83 @@
...
+
+    ```Python hl_lines="8"
+    {!> ../../../docs_src/path_params_numeric_validations/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/path-params-numeric-validations.md:81]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/path-params-numeric-validations.md [hunk 31]
```diff
@@ -43,9 +107,14 @@
 
 因此，你可以将函数声明为：
 
-```Python hl_lines="7"
-{!../../../docs_src/path_params_numeric_validations/tutorial002.py!}
-```
+=== "Python 3.6 non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/path-params-numeric-validations.md:113]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 32]
```diff
@@ -33,9 +49,41 @@
...
+        Prefer to use the `Annotated` version if possible.
+
+    ```Python hl_lines="18-23"
+    {!> ../../../docs_src/schema_extra_example/tutorial003_py310.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/schema-extra-example.md:82]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/schema-extra-example.md [hunk 33]
```diff
@@ -33,9 +49,41 @@
...
+=== "Python 3.6+"
+
+    ```Python hl_lines="23-28"
+    {!> ../../../docs_src/schema_extra_example/tutorial003_an.py!}
+    ```
+
+=== "Python 3.10+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/schema-extra-example.md:73]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

File: docs/zh/docs/tutorial/security/first-steps.md [hunk 34]
```diff
@@ -20,9 +20,26 @@
...
+=== "Python 3.6+"
+
+    ```Python
+    {!> ../../../docs_src/security/tutorial001_an.py!}
+    ```
+
+=== "Python 3.6+ non-Annotated"
+
+    !!! tip
+        Prefer to use the `Annotated` version if possible.
```
Comment (mkdir700) [inline docs/zh/docs/tutorial/security/first-steps.md:38]:
```suggestion
        尽可能选择使用 `Annotated` 的版本。
```

**Review by mkdir700 — COMMENTED** [review]
Some sentences have not been translated.

**Comment by tiangolo** [comment]
📝 Docs preview for commit 66e8d3d3488ea2e7eed20c5984fca4809d39fb2c at: https://64a78a5bb0058521e7475fd5--fastapi.netlify.app

**Comment by mahone3297** [comment]
Is there anybody to approve this PR? Do I need to follow any procedures?
Two native speaker have reviewed it and I have changed the code based on the review.

**Review by wdh99 — APPROVED** [review]

**Review by Slijeff — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thank you @mahone3297! 🚀 

And thanks for the reviews @mkdir700, @Slijeff, @wdh99 🙇

**Pull request merged**